src/helpers/game_phases.h

```diff
@@ -65,12 +65,11 @@ constexpr void subPhaseScore(evaluation::gamephase::Score& score, const std::arr
  * using precomputed PESTO piece-square tables.
  * Also the material score (game phase dependent) is increased whenever probed
  */
-constexpr void addPestoPhaseScore(evaluation::gamephase::Score& score, Piece piece, BoardPosition pos)
+constexpr __attribute__((always_inline)) void addPestoPhaseScore(evaluation::gamephase::Score& score, Piece piece, BoardPosition pos)
```

Comment:
I'm confused why this is necessary. constexpr should implicitly be inline. Was this tested with release build as well?